src/client/render2d/canvas/text.ts

```diff
@@ -0,0 +1,37 @@
+import { Transform } from '../../math/transform'
+import { Vector2 } from '../../math/vector2'
+import { TextGeometry } from '../geometry/text_geometry'
+import { Shape } from '../object/shape'
+
+import { applyAppearance } from './util'
```

Comment:
How about `rendering` instead of `util`? Or some other word that describes the things that go in here.